src/content/developers/tutorials/Using-UseDApp-with-unity/index.md

```diff
@@ -0,0 +1,206 @@
+
+### 3\. Write code to receive the balance {#unity-balance}
+In asset folder create a new cs file ```BalanceController.cs```, drag and drop this file into gameobject ```BalanceText```,open the code file and writer the code below.
+```csharp
+    void EtherBalanceToUnity(string amount)
+    {
+        GameObject.Find("BalanceText").GetComponent<Text>().text = $"Balance: {amoun} EHT";
+    }
+```
+The **EtherBalanceToUnity** will receive the account balance from react app.
+
+### 4\. Write code to send a transaction {#unity-send-transaction}
+In asset folder create a new folder with name **Plugins**, create a new file ```script.jslib``` inside this new folder, 
+[jslib](https://docs.unity3d.com/Manual/webgl-interactingwithbrowserscripting.html) script are similar  to javascript,
+this script build a bridge between your Unity project and the react app,**SendTranscationFromUnitya** is the function that will exist in both unity and react app,write the code below.
+```javascript
+    mergeInto(LibraryManager.library, {
+      SendTranscationFromUnity:function (address,amount) {
+        ReactUnityWebGL.SendTranscationFromUnity(Pointer_stringify(address), amount);
+      },
+});
+```
+Create a new file ```SendController.cs``` put the code in the gameobject **Button**,open the file and add this code.
+```csharp
+    //The variable that stores the input field.
+    public InputField input;
+    //Tell to unity to use the script.jslib in Plugins folder, to use the SendTranscationFromUnity function in script.jslib . 
+    [DllImport("__Internal")]
+    private static extern void SendTranscationFromUnity(string address, double amount);
+    void Start()
+    {
+        //Add a method to send information about your transaction.
+        GameObject.Find("Button").GetComponent<Button>().onClick.AddListener(Transaction);
+        //Set the input field.
+        input = GameObject.Find("InputField").GetComponent<InputField>();
+        //Since unity Webgl does not support ctrl-v without external code, set demo address here.
+        input.text = "0xf8331f18a7106bF6B9C0847b3BbC5B6180806A2C";
+    }
+    void Transaction()
+    {
+        //A valid eth address is 42 characters long 
+        if (input.text.Length == 42)
+        {
+          //Send transaction information to react app.
+          SendTranscationFromUnity(input.text, 1e15);
+        }
+    }
+```
+
+Now our unit project is ready, wait for the next tutorial part to build the project, the code only works when is builded, it will not work in unity editor mode, 
+to learning more about unity in web browser and how interact using javascript consult the [official documentation](https://docs.unity3d.com/Manual/webgl-gettingstarted.html)
+
+## React app with UseApp {#React-UseDApp}
+Create a react app that use UseDApp,with UseDapp is possible easy connect you web wallet in a dapp to send transaction and get information about your account,
```

Comment:
We should break this into multiple sentences as comma use is confusing and hard to follow